app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryDescription.js

```diff
@@ -36,19 +36,22 @@ class ExpandedStoryDescription extends React.Component {
   };
 
   descriptionTextArea(description) {
-    const { onEdit } = this.props;
+    const { onEdit, readOnly } = this.props;
```

Comment:
Why is it called `readOnly` for his one instead of `disabled`?

Reply:
It uses the readonly attribute for the input field so the user can copy the text. We could use disabled as well if that is not really necessary.

Reply:
We can have the component to be called `disabled` but pass it as the value of `readOnly` of the `textarea`:

```js
<textarea
  readOnly={ disabled }
```